### ISSUE TITLE
Use Cube Store

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#2343]
https://cube.dev/blog/introducing-cubestore/

**Comment by ERAMANrepartos** [comment]
Did this happen because of the universal export? @lucasferraro

**Comment by lucasferraro** [comment]
> Did this happen because of the universal export? @lucasferraro

No, seems like Cube Store is something to improve performant to the existing implementation of Cube. @alexsegura ?

**Comment by alexsegura** [comment]
Cube Store is actually used in production, not in dev. 
Not so much relevant anymore.
